src/picasso-components/react-components/navigation-panel.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { IconButton } from '@material-ui/core';
+import SVGIcon from './icons/SVGIcon';
+import NUMBERS from '../../constants/numbers';
+import ICONS from './icons';
+
+const buttonWidth = NUMBERS.NAVIGATION_PANEL.GRID_WIDTH;
+const padding = NUMBERS.NAVIGATION_PANEL.PADDING;
+
+const useStyles = (rtl) =>
+  makeStyles(() => {
+    const horizontalDistance = rtl ? 'left' : 'right';
+    return {
+      onFocus: {
+        '&:focus': {
```

Comment:
I think add focus style inside common should work too

Reply:
good idea.